static/js/public/snap-details/screenshots.js

```diff
@@ -13,7 +13,14 @@ export default function initScreenshots(screenshotsId) {
     const url = event.target.src;
 
     if (url) {
-      lightbox.openLightbox(url, images);
+      // prev.offsetWidth and prev.offsetHeight are used to detect the visibility
+      // of the prev element.
+      if (prev.offsetWidth === 0 && prev.offsetHeight === 0) {
```

Comment:
I've written the code that hides prev/next buttons on mobile, but it took me quite a while to get what is going on here.

Reply:
I'd rather find a better way to detect it via an element's visibility, rather than setting numbers - perhaps whether the mobile menu is visible?

I can definitely make it as a function so it can be used in other places too in the future.